src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -857,6 +865,17 @@ private function createVersion(ContainerBuilder $container, $version, $format, $
         return new Reference('assets._version_'.$name);
     }
 
+    private function createJsonManifestVersion(ContainerBuilder $container, $manifestPath, $name)
+    {
+        $def = new ChildDefinition('assets.json_manifest_version_strategy');
+        $def
+            ->replaceArgument(0, $manifestPath)
```

Comment:
I agree, but I copied this code from the above method, which is not private. So, might there be a reason for it to be public? https://github.com/symfony/symfony/blob/4927993835457bc2ae23826f5572f97e26dda0ec/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php#L843-L856

I'm going to try making it private.